app/views/steps/case/charges/edit.html.erb

```diff
@@ -10,6 +10,16 @@
     <%= step_form @form_object do |f| %>
       <%= f.govuk_text_field :offence_name, autocomplete: 'off' %>
 
+      <%= f.fields_for :offence_dates do |od| %>
+        <%= od.govuk_date_field :date, maxlength_enabled: true, legend: { size: 'm' } %>
```

Comment:
We might have to wrap each of the date field in a `govuk_fieldset`, not sure if that was purely needed for the validation and thus proper linking or maybe was just for styling etc. We will figure out next with validations, no worries for now!